config/crd/bases/operator.ibm.com_commonservices.yaml

```diff
@@ -4,6 +4,8 @@ kind: CustomResourceDefinition
 metadata:
   annotations:
     controller-gen.kubebuilder.io/version: v0.14.0
+  labels:
+    foundationservices.cloudpak.ibm.com: commonservice
```

Comment:
we use `foundationservices.cloudpak.ibm.com: crd` as a label for the crd in existing docs https://www.ibm.com/docs/en/cloud-paks/foundational-services/4.7?topic=fsbr-cloud-pak-foundational-services-backup-restore-clusters-single-instance-foundational-services

Reply:
We could technically use this instead since our command in our docs specifies `--overwrite` and/or we could update our docs/restore resources but going strictly on what's already in place, we would prefer crd instead of commonservice

Reply:
If you prefer that though, I don't mind. Restoring any of the CRDs shouldn't be a problem as they're all linked to the subscription version (in principle).

Reply:
I'd vote the CR name as that's what we do for all the intregration CRs

Reply:
Let me test what we discussed in https://github.ibm.com/IBMPrivateCloud/roadmap/issues/63820. If we can include them in the same restore then it shouldn't matter that the labels are the same (and we may update our process to reflect that). Otherwise I would prefer `crd`

Reply:
@bluzarraga ping on this one, did you test it?

Reply:
Yes, it does work but after discussing with the rest of the team we would prefer the label be `crd` instead of `commonservice`

Reply:
PR updated to use CRD.

I'd be interested to know why you chose to go with crd instead of commonservice, but feel free to merge when ready (and please label for backport to SC2.)